src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+public class RightTriangle extends Shape {
+    private double side;
+    private double base;
+
+    public RightTriangle(String color, double side, double base) {
+        super(Colors.valueOf(color));
+        this.side = Math.round(Math.abs(side));
+        this.base = Math.round(Math.abs(base));
+    }
+
+    public double getSide() {
+        return side;
+    }
+
+    public void setSide(double side) {
+        this.side = side;
+    }
+
+    public double getBase() {
+        return base;
+    }
+
+    public void setBase(double base) {
+        this.base = base;
```

Comment:
Consider validating the `base` value in the `setBase` method to ensure it is non-negative, similar to the constructor.